.github/workflows/release.yaml

```diff
@@ -126,8 +175,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   release-macos:
     runs-on: macos-latest
-    strategy:
-      fail-fast: true
```

Comment:
Any reason you're removing this?

Reply:
Yes, because it is invalid syntax without the matrix keyword.